jpa/README.md

```diff
@@ -214,7 +214,7 @@ Once the new pod has initialized you can see the secret database connection data
 ```shell
 > odo exec -- bash -c 'export | grep DATABASE'
 declare -x DATABASE_CLUSTERIP="172.30.36.67"
-declare -x DATABASE_DB_NAME="sampledb"
+declare -x DATABASE_DB_NAME="sampledatabase"
```

Comment:
@ajm01 did odo team build any doc off this name?  If not maybe just leave it alone?

Reply:
I believe they are in process, not finalized as of yet - I can check with them

Reply:
I will let Andy answer your question before doing anything, but let me just mention that the instructions explicitly state that the database name should sampledatabase:
```
- Click on the 'Create Database' button
- Change the value in the Database Name field to 'sampledatabase'
- Change the value in the Database User field to 'sampleuser'
- Change the value in the Database Password field to 'samplepwd'
```
but the sample outputs show: sampledb as the database name.

Reply:
I think it is fine to standardize it here to one name and I can let them know there was an update to line everything up in the doc.